Modules/Main.lua

```diff
@@ -371,7 +375,35 @@ function main:CallMode(func, ...)
 	end
 end
 
+function main:LoadPastebinBuild()
+	if (arg[1] == nil or string.sub(arg[1], 1, 6) ~= "pob://" ) then
+		return false
+	end
+
+	local pastebinCode = string.sub(arg[1], 7, string.len(arg[1]))
+	if (string.sub(pastebinCode, -1) == "/") then
+		pastebinCode = string.sub(pastebinCode, 1, string.len(pastebinCode) - 1)
+	end
+	launch:DownloadPage("https://pastebin.com/raw/" .. pastebinCode, function(page, errMsg)
```

Comment:
Consider automatically driving the import tab instead of duplicating its logic here.